Four Brains/Model/K.swift

```diff
@@ -9,24 +9,110 @@
 import Foundation
 
 class K {
-    class MUTECONSTANTS {
+    class MuteConstants {
```

Comment:
Nice. Not sure on swift standards for naming variables, but this is easier for me to read

Reply:
Thanks! Camel case is the standard for this usage